google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStubSettings.java

```diff
@@ -433,6 +443,7 @@ public Builder toBuilder() {
      */
     private Builder() {
       this.appProfileId = SERVER_DEFAULT_APP_PROFILE_ID;
+      this.isRefreshingChannel = false;
```

Comment:
Please update `Builder(EnhancedBigtableStubSettings settings)` to respect this flag as well

Reply:
actually nvm, you already did